Dockerfile

```diff
@@ -1,9 +1,17 @@
-From tomcat:8.5-alpine
+FROM maven:3.6.3 as maven-build
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+RUN mvn package -DskipTests
+
+FROM tomcat:8.5-alpine
 # VOLUME /tmp
 RUN /bin/rm -fR /usr/local/tomcat/webapps/*
-COPY target/sesar-rest-api-1.0.0.war /usr/local/tomcat/webapps/api.war
+RUN sed -i 's/8080/80/' /usr/local/tomcat/conf/server.xml
+WORKDIR /usr/local/tomcat/webapps/
+COPY --from=maven-build /usr/src/app/target/sesar-rest-api-1.0.0.war /usr/local/tomcat/webapps/api.war
 RUN sh -c 'touch /usr/local/tomcat/webapps/api.war'
```

Comment:
Not urgent, but running as root is insecure. Don't change now, but we should make a note to update this and SESAR

Example:

```
FROM python:3.9-alpine
RUN addgroup -S appgroup \

&& adduser -S appuser -G appgroup


USER appuser
[REST_OF_YOUR_DOCKERFILE]
```